README.md

```diff
@@ -9,10 +9,7 @@ GlusterD, while also becoming more modular and easing extensibility.
 
 ## Documentation
 
-* [Quick Start User Guide](doc/quick-start-user-guide.md)
-* [Development Guide](doc/development-guide.md)
-* [Coding Guidelines](doc/coding.md)
-* [REST API Reference](doc/endpoints.md)
+Complete documentation for users and developers can be found [here](doc/index.md)
```

Comment:
IMO Its better to point out User Documentation and Developer Documentation inside Readme.md itself.

Reply:
Good point. Will change :+1:

Reply:
@rishubhjain Done